src/debuginfo.cpp

```diff
@@ -207,6 +207,15 @@ struct revcomp {
     { return lhs>rhs; }
 };
 
+#ifdef LLVM38
+struct strrefcomp {
+    bool operator() (const StringRef& lhs, const StringRef& rhs) const
+    {
+        return lhs.compare(rhs) > 0;
```

Comment:
Ooops, the order was copied from the `revcomp` above and is reversed... Shouldn't matter so I'll just leave it like this...